packages/core/examples/toastExample.tsx

```diff
@@ -32,11 +32,13 @@ export class ToastExample extends BaseExample<IToasterProps> {
 
     private TOAST_BUILDERS: IToastDemo[] = [{
         action: {
+            href: "https://www.google.com/search?q=toast&source=lnms&tbm=isch",
+            target: "_blank",
```

Comment:
Is this optional or will it always open in a new tab? Prefer to give users the option but not be opinionated